custom-namespaces.yaml

```diff
@@ -732,6 +739,10 @@ firefox_accounts:
       type: table_explore
       views:
         base_view: health_issues
+    health_issues:
```

Comment:
There's already a `health_issues` explore defined above.
```suggestion
    health_lighthouse:
```

Reply:
fixed in new push. thanks